rescan.py

```
@@ -446,81 +488,93 @@ def run_scan():
-        print("Please check your Plex server URL and token")
-        exit(1)
+        os.makedirs(os.path.dirname(default_config_path), exist_ok=True)
+        with open(default_config_path, 'w', encoding='utf-8') as f:
+            yaml.dump(default_settings, f, indent=4, default_flow_style=False, sort_keys=False)
+        print("✅ Default configuration created. Please edit it with your details and restart.")
+    except IOError as e:
+        print(f"❌ Failed to write default configuration file: {e}")
+    
+    return None
 
 def main():
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)